styles/card/card.css

```diff
@@ -0,0 +1,80 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: white;
+  --secondary-color: rgb(15, 1, 1);
+  --default: rgb(53, 52, 52);
+}
+
+.card-container{
+    padding: 1rem;
+}
+
+.card {
+  color: white;
+  width: 16rem;
+  padding: 1rem;
+  background-color: var(--secondary-color);
+}
+
+.card-title {
+  margin: 8px 0;
+  font-size: 1.4rem;
+  font-weight: bold;
+}
+
+.card-btn {
+  text-transform: uppercase;
+  font-weight: bold;
+  padding: 9px;
+  margin: 2px;
+  cursor: pointer;
+}
+.card-logo {
+  height: 8rem;
+  width: 10rem;
+  
+}
+
+.card-price{
+  font-weight: bold;
+  font-size: 1.2rem;
+}
+
+.card-desc{
+  margin: 1rem 0;
+}
+
+.card-vertical{
+  background-color: white;
```

Comment:
Use variables for colors,

Reply:
sure

Reply:
will refactor with comments in all components